socorro/unittest/processor/test_mozilla_transform_rules.py

```diff
@@ -1832,3 +1594,70 @@ def test_with_malformed_addons_field(self):
             'elemhidehelper@adblockplus.org:1.2.1',
         ]
         assert processed_crash.addons == expected_addon_list
+
+
+class TestSignatureGeneratorRule:
+    def test_empty_raw_and_processed_crashes(self):
+        rule = SignatureGeneratorRule(get_basic_config())
+        raw_crash = {}
+        processed_crash = {}
+        processor_meta = get_basic_processor_meta()
+
+        ret = rule._action(raw_crash, {}, processed_crash, processor_meta)
```

Comment:
What does `ret` mean as a variable name?

Reply:
"return" is a reserved word, so we can't use that. So I tend to abbreviate it to "ret". It's not a great name. I'd be game for alternatives if you have one.

Reply:
I see. I think, in the past, I have used `result`.